testing/src/test/java/org/oppia/android/testing/robolectric/OppiaShadowActivityManagerTest.kt

```diff
@@ -0,0 +1,152 @@
+package org.oppia.android.testing.robolectric
+
+import android.app.ActivityManager
+import android.app.Application
+import android.content.Context
+import android.os.Build
+import android.os.Debug
+import androidx.test.core.app.ApplicationProvider
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import com.google.common.truth.Truth.assertThat
+import dagger.BindsInstance
+import dagger.Component
+import dagger.Module
+import dagger.Provides
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.annotation.Config
+import org.robolectric.annotation.LooperMode
+import javax.inject.Singleton
+
+private val oppiaShadowActivityManager by lazy { OppiaShadowActivityManager() }
```

Comment:
Why use this vs. creating the shadow through Robolectric?

Reply:
done.